src/tools/cephfs/cephfs-shell

```diff
@@ -401,10 +414,19 @@ class CephFSShell(Cmd):
 
     class path_to_bytes(argparse.Action):
         def __call__(self, parser, namespace, values, option_string=None):
-            if isinstance(values, str):
-                values = to_bytes(values)
-            if isinstance(values, list):
-                values = list(map(to_bytes, values))
+            values = to_bytes(values)
+            setattr(namespace, self.dest, values)
+
+    class get_list_of_bytes_path(argparse.Action):
```

Comment:
Why did you add this class?

Reply:
The extra code in here might disturb other methods using `path_to_bytes`.  I'll do that on a PR right after this one merges.